src/components/app/ListOfPublishedProfiles.css

```diff
@@ -53,29 +37,3 @@
 .publishedProfilesLink:hover {
   background: var(--grey-90-a05);
 }
-
-/* In larger screens, we remove the flex characteristics but force some width
- * for the first element.
- * We also add some more padding and change how the width is computed.
- *
- * 750px and 465px, seemed like the right values when testing on Linux. Fonts
- * are usually larger on Linux for the profiler, so in other OSes we could
- * decide that the value is too large, but this is fine. */
-@media (min-width: 750px) {
```

Comment:
We don't have this `min-width: 750px` media rule anymore and we don't have its comment either. Is this intended?

Reply:
Good find, I think I removed it by mistake indeed... I'll double check what the behavior is with / without this media.

Reply:
Thanks for your eagle eyes! I added it back, this was definitely missing. (as well as the comment)